lib/int.fz

```diff
@@ -108,18 +108,9 @@ is
   infix ** (other int) int
   pre other >= zero
   is
-    if other = zero
-      one
-    else if other = one
-      thiz
-    else
-      is_even := other.n.bits.last = false
-      if is_even
-        tmp := thiz**(other / int 2)
-        tmp * tmp
-      else
-        thiz * (thiz**(other-one))
-
+    match s
+      plus => int plus (n ** other.n)
+      minus => int minus (n ** other.n)
```

Comment:
This seems wrong: -1 ** 2 should be +1.

Reply:
the test also fails but I didn't notice... :-(

sam@debian ~/n/fuzion (lib_uint_init_and_exponentation)1$ fz-c tests/int/int_test.fz |grep FAIL
FAILED: int: m -5 ** n 0
FAILED: int: m -5 ** n 2
FAILED: int: m -5 ** n 4
FAILED: int: m -4 ** n 0
FAILED: int: m -4 ** n 2
FAILED: int: m -4 ** n 4
FAILED: int: m -3 ** n 0
FAILED: int: m -3 ** n 2
FAILED: int: m -3 ** n 4
FAILED: int: m -2 ** n 0
FAILED: int: m -2 ** n 2
FAILED: int: m -2 ** n 4
FAILED: int: m -1 ** n 0
FAILED: int: m -1 ** n 2
FAILED: int: m -1 ** n 4